CONTRIBUTING.md

```diff
@@ -110,19 +116,21 @@ Please run ESLint and Prettier with these commands so you're good on the CI proc
 
 ```sh
 $ npm run lint
-$ npm run format
+$ npm run prettier
 ```
 
 And you're set!
 
 ### NPM scripts
 
-- `npm install` - Your typical 5 million packages installation
-- `npm run build` - Node won't automagically run Typescript files
+This repo is using npm@v7 workspaces so you can add `-w backend` or `-w frontend` to each of these commands.
+
+- `npm install` - Your typical 5 million packages installation.
+- `npm run build` - Node won't automagically run Typescript files.
 - `npm run start` - Start the fun stuff, make sure you've run the build step before running this.
-- `npm run dev` - When you want stuff to be automated
-- `npm run test` - Make sure everything works correctly
-- `npm run lint` - Check the files from your silly mistakes
-- `npm run format` - You like pretty formatted code, right?
-- `npm run docker:build` - Prepare for 'it works on my machine' solution
-- `npm run docker:run` - 'it works on my machine' is no more
+- `npm run dev` - When you want stuff to be automagically reloaded.
+- `npm run test` - Make sure everything works correctly. Only works for backend workspaces.
+- `npm run lint` - Check the files from your silly mistakes.
+- `npm run prettier` - You like pretty formatted code, right?
+- `npm run docker:build` - Prepare for 'it works on my machine' solution. Only works for backend workspaces.
+- `npm run docker:run` - 'it works on my machine' is no more. Only works for backend workspaces.
```

Comment:
this both should be moved on a separate list I guess, because they lives on the main package.json